src/Auth0.ManagementApi/Clients/BlacklistedTokensClient.cs

```diff
@@ -22,7 +22,7 @@ public BlacklistedTokensClient(IApiConnection connection)
         /// <summary>
         /// Gets all the blacklisted claims.
         /// </summary>
-        /// <param name="aud">The aud claim for which you want to get blacklisted tokens. This is your API Key.</param>
+        /// <param name="aud">The JWT's aud claim. The client_id of the client for which it was issued.</param>
```

Comment:
> The client_id of the client for which it was issued.

Consider:

> This Client ID of the Application for which it was issued.

... though I'm not sure that's 100% true. The Client ID as audience is for ID tokens, which you don't blacklist. I realize that this is straight from the docs but we might want to chase getting that updated.